EPFL-Library-Plugins.php

```diff
@@ -139,4 +139,22 @@ function epfl_library_beast_redirect_process_shortcode($attributes, $content = n
             </script>';
 }
 add_shortcode('epfl_library_beast_redirect', 'epfl_library_beast_redirect_process_shortcode');
+
+
+function insert_beastbox() {
+    $request_uri = $_SERVER['REQUEST_URI'];
+	// Specific Custom Fields
+    $hide_beastbox = get_post_meta( get_the_ID(), 'hide_beastbox', true );
+
+	if (( $hide_beastbox != "1" ) && (!(strpos($request_uri, "wp-admin")))) {
+		if (strpos($request_uri, "/fr/")) {
```

Comment:
Pour contrôler quelle langue c'est, tu peux utiliser la fonction `pll_current_language` fournie par le plugin de gestion des langues. La doc est sur cette page: https://polylang.wordpress.com/documentation/documentation-for-developers/functions-reference/
Et afin d'éviter une crash si le plugin de gestion des langues n'est pas dispo, faut juste faire un check si la fonction `pll_current_language` existe avant de l'appeler https://www.php.net/manual/fr/function.function-exists.php

Reply:
Merci pour tes remarques. En principe tout devrait être corrigé.

Reply:
Top! j'ai mergé. ça devrait du coup passer en production dans une semaine pile.